data/facts.js

```diff
@@ -2,16 +2,16 @@ import { shuffle } from '@utils';
-  'The City of Los Angeles has a DataLA Publication that showcases data stories and insights from the Mayor’s Data Team and other contributors.',
-  'The City of Los Angeles has interactive dashboards that allow users to explore the city budget, crime trends, storm water capture and other topics.',
-  'The City of Los Angeles has ambitious storm water capture goals and uses data to monitor and improve its performance.',
-  'The City of Los Angeles has a Privacy and Security Task Force that works on ensuring contactless government services are safe and accessible.',
-  'The City of Los Angeles has conducted an equity procurement analysis to assess how city contracts are distributed among different groups of vendors.',
-  'The City of Los Angeles has used location-based data to study the spatial distribution of Covid-19 cases and identify hotspots and vulnerable areas.',
-  'The City of Los Angeles has supported the mapping of black-owned businesses in the city to promote economic inclusion and diversity.',
-]);
-
-export default facts;
+  'The LA Neighborhood Council system comprises 99 Neighborhood Councils, each serving approximately 40,000 people.',
+  'In some cases, the boundaries of two or more Certified Neighborhood Councils may overlap if the proposed area is designed for public use, such as a park, school, library, police or fire station, or major thoroughfare; or houses a historical landmark or facility.', 
+  'The Neighborhood Council system was launched in 1999 to ensure that the City government stays connected and adaptive to the unique needs and diverse characteristics of LA’s various communities, promoting more inclusive and representative local governance.',
+  'Hack for LA is a volunteer-driven civic tech organization and is a project of Civic Tech Structure, Inc.—an organization that empowers nonprofits and governments through innovative solutions, scaling, and experimentation, to drive positive change and community impact.', 
+  'Spanning 4,084 square miles, LA County is one of the largest in the United States.', 
+  'With nearly 10 million residents—accounting for 27% of California’s total population—LA County is the most populous of any county in the United States.', 
+  'LA County encompasses 88 cities and over 120 unincorporated communities.',
+  'Neighborhood Councils are funded by taxpayer dollars, and their board members, who are elected by local communities, serve as volunteer LA City government officials.',
+  'Each Neighborhood Council is distinct, with its own board structure and stakeholder representation. Board sizes range from 7 to 35 members, with terms typically lasting two years.',
+  'Neighborhood Councils receive $32,000 annually in public funds. These funds support community events, programs, and advocacy initiatives, addressing issues such as infrastructure, public safety, and economic development.',
```

Comment:
This should be with a '$' sign as '$29497.15'. The rest looks good to me. Pulled down and tested, all the new fun facts showed up as expected.